corehq/pillows/groups_to_user.py

```diff
@@ -43,6 +45,26 @@ def get_group_to_user_pillow(pillow_id='GroupToUserPillow', num_processes=1, pro
         ),
     )
 
+def get_group_es_pillow(pillow_id='GroupESPillow', num_processes=1, process_num=0, **kwargs):
```

Comment:
rename to `get_group_pillow` and pillow_id to `group_pillow`

Reply:
Same as above. This name is already in use by old pillow, I will see what we can do